Tasks/PublishSymbols/PublishSymbolsTask.ps1

```diff
@@ -0,0 +1,44 @@
+[CmdletBinding()]
+param()
+
+Trace-VstsEnteringInvocation $MyInvocation
+try {
+    $symbolServerType = Get-VstsInput -Name 'SymbolServerType' -Require
+
+    if ($SymbolServerType -eq "FileShare")
+    {
+        & "$PSScriptRoot\PublishSymbols.ps1"
+    }
+    elseif ($symbolServerType -eq "TeamServices")
```

Comment:
Have you considered adding a visibility rule on the input to hide it when TeamServices is selected?